magicbus/base.py

```diff
@@ -166,7 +166,7 @@ def __init__(self, transitions=None, errors=None,
             self.listeners[c] = set()
 
         if id is None:
-            id = hex(random.randint(0, sys.maxint))[-8:]
+            id = hex(random.randint(0, sys.maxsize))[-8:]
```

Comment:
Side note: looks like it's "work for uuid module". But not in this PR, of course :)

Reply:
Also: would this work?
```suggestion
            id = hex(random.randint(0, float('inf')))[-8:]
```